apps/web/src/routes/_content/courses/index.tsx

```diff
@@ -70,8 +71,30 @@ const CourseInfoSection = ({ course }: { course: JoinedCourse }) => {
 };
 
 export const CourseCard = ({ course }: { course: JoinedCourse }) => {
+  const { data: reviews } = trpc.content.getPublicCourseReviews.useQuery(
```

Comment:
it's making a request each time we change filters -> we need to find another way to avoid that